MVVMRxSwiftDemo/Sources/Presentation/Modules/SurveyQuestion/Cell/QuestionCollectionCellViewModel.swift

```diff
@@ -27,9 +28,26 @@ final class QuestionCollectionCellViewModel: QuestionCollectionCellViewModelProt
     var output: QuestionCollectionCellViewModelOutput { self }
     
     let questionText: Driver<String>
+    let answerField: Driver<AnswerField>
     
-    init(questionText: String) {
+    init(questionText: String, displayType: DisplayType, pickType: PickType, answers: [SurveyAnswer]) {
         self.questionText = Driver.just(questionText)
+        
+        switch displayType {
+        case .star, .heart, .smiley:
+            answerField = Driver.just(RatingAnswerField(displayType: displayType, answers: answers))
+        case .nps:
+            answerField = Driver.just(NPSAnswerField(answers: answers))
+        case .textarea, .textfield:
+            answerField = Driver.just(TextFieldAnswerField(displayType: displayType, answers: answers))
+        default:
```

Comment:
I would prefer to define all cases instead of using `default`, because when we adding a new type of question, the Xcode will show an error here for us to update the new case.
```suggestion
        case .choice, .intro, .outro:
```